crates/control/src/behavior/stand.rs

```diff
@@ -14,6 +15,26 @@ pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
                 },
             })
         }
+        PrimaryState::Playing => {
+            if (matches!(
+                world_state.game_controller_state,
+                Some(GameControllerState {
+                    game_phase: GamePhase::PenaltyShootout { .. },
+                    ..
+                })
+            ) && world_state.robot.role == Role::Striker
+                && world_state.ball.is_none())
+            {
+                let robot_to_field = world_state.robot.robot_to_field?;
+                Some(MotionCommand::Stand {
+                    head: HeadMotion::LookAt {
+                        target: robot_to_field.inverse() * Point2::origin(),
```

Comment:
I replaced the HeadMotion::LookAt with HeadMotion::Center, so that the robot is looking forward. This means in case of a penalty shootout the robot will now look at the goal and penalty spot in front of him.